Scripts/Runtime/Entities/Util/WorldUtil.cs

```diff
@@ -0,0 +1,230 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Anvil.CSharp.Logging;
+using Unity.Entities;
+using UnityEngine;
+using UnityEngine.LowLevel;
+using UnityEngine.PlayerLoop;
+
+namespace Anvil.Unity.DOTS.Entities
+{
+    /// <summary>
+    /// A system group to house a <see cref="World"/>s <see cref="EndInitializationEntityCommandBufferSystem"/>.
+    /// Used for mutli-world playerloop optimization.
+    /// </summary>
+    /// <remarks>Kept outside <see cref="WorldUtil"/> so their names read better in the editor.</remarks>
+    [DisableAutoCreation]
+    internal class EndInitializationCommandBufferSystemGroup : ComponentSystemGroup { }
+
+    /// <summary>
+    /// A system group to house a <see cref="World"/>s <see cref="EndSimulationEntityCommandBufferSystem"/>.
+    /// Used for mutli-world playerloop optimization.
+    /// </summary>
+    /// <remarks>Kept outside <see cref="WorldUtil"/> so their names read better in the editor.</remarks>
+    [DisableAutoCreation]
+    internal class EndSimulationCommandBufferSystemGroup : ComponentSystemGroup { }
```

Comment:
Minor but I'm wondering if there's value in having these names have something in them to denote that they are custom? Or at least not Unity SystemGroups? 

Right now if I saw these in the Entities Debugger or Profiler, I would think they were default Unity groups. 

Maybe `AnvilEndInitializationCommandBufferSystemGroup` etc?

Reply:
Good point. I added the `_Anvil` as a suffix to all of these.
That way it's easy to identify as non-default but you don't constantly have to parse the word Anvil when hunting for things in the `PlayerLoop` list. Much easier to read and recognize the group for what it really is.

Reply:
Cool perfect